python/ray/tests/test_webui.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray
+
+
+def test_get_webui():
+    addresses = ray.init(include_webui=True)
+    webui_url = addresses['webui_url']
```

Comment:
```suggestion
    webui_url = addresses["webui_url"]
```

The linter will complain about this.

Also, will this test work? To work, it needs to have Python 3, psutil, and aiohttp installed in Travis.